core/trace.c

```diff
@@ -454,7 +435,8 @@ void tracepoint_schedule(trigger_t* trigger, interval_t extra_delay) {
             && trigger->reactions[0] != NULL) {
         reactor = trigger->reactions[0]->self;
     }
-    tracepoint(schedule_called, reactor, 0, 0, NULL, trigger, extra_delay);
+    // FIXME: First 0 should be the worker.
```

Comment:
Maybe use -1 so that it does not look like a valid worker number? It looks pretty difficult to provide the correct worker number here, but maybe we can document it as an issue.

Reply:
Let's resolve this before merging and not leave the FIXME.

Reply:
This has been resolved with commit #684e354